spec/helpers/register.rb

```diff
@@ -0,0 +1,16 @@
+def register(
+             name: 'Tristan',
+             username: 'elchapo91',
+             email: 'elchapo91@email.com',
+             password: '12345',
+             password_confirmation: '12345'
+             )
```

Comment:
Style/ClosingParenthesisIndentation: Indent `)` the same as the start of the line where `(` is.